src/cms-content/footer/index.ts

```diff
@@ -1,4 +1,5 @@
 import { Markdown } from '../utils';
+import { shuffle } from 'lodash';
```

Comment:
It's better to import only the function that we need performance-wise (we can install a plugin to do that automatically, but that's a bit of a hassle)

```ts
import shuffle from 'lodash/shuffle';
```